src/server/enrichment-map/enrichment/enrichment.js

```diff
@@ -74,12 +74,21 @@ const enrichment = (query, userSetting) => {
       const tNameIndex = 11;
       const tDepthIndex = 12;
       const QTListIndex = 13;
+      ret.orderedQuery = formData.ordered_query;
+      ret.userThr = formData.user_thr;
+      ret.minSetSize = formData.min_set_size;
+      ret.maxSetSize = formData.max_set_size;
+      ret.thresholdAlgo = formData.threshold_algo;
+      ret.custbg = formData.custbg;
+      ret.custbgCb = formData.custbg_cb;
```

Comment:
Shouldn't we just remove `custbgCb` as an exposed option and set this (boolean) internally based upon whether `custbg` is null or not?

>custbg_cb - [bool] Use custom statistical background (g:GOSt, g:Cocoa)
custbg - [string] A space-separated list of symbols to use as custom stat. background (g:GOSt, g:Cocoa)

Reply:
In fact, this is the way it works! I entered a background list in custbg and turned the custbgCB to 1 and 0 and it didn't make a difference.

Reply:
That makes sense to me, but...is the option custbgCb useless? Why does it still exist if decisions are made purely based on custbg?

Reply:
I tried a call directly to g:Profiler and custbg_cb didn't make any difference. Remove.